frontend/src/__tests__/integration/pages/acceleratorProfiles/AcceleratorProfiles.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { StoryFn, Meta, StoryObj } from '@storybook/react';
+import { rest } from 'msw';
+import { mockAcceleratorProfile } from '~/__mocks__/mockAcceleratorProfile';
+import { mockProjectK8sResource } from '~/__mocks__/mockProjectK8sResource';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import useDetectUser from '~/utilities/useDetectUser';
+import { mockStatus } from '~/__mocks__/mockStatus';
+
+import AcceleratorProfiles from '~/pages/acceleratorProfiles/AcceleratorProfiles';
+
+export default {
+  component: AcceleratorProfiles,
+  parameters: {
+    msw: {
+      handlers: [
+        rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+          res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
+        ),
+        rest.get('/api/status', (req, res, ctx) => res(ctx.json(mockStatus()))),
+        rest.get(
+          '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles',
+          (req, res, ctx) => res(ctx.json(mockK8sResourceList([mockAcceleratorProfile()]))),
+        ),
+      ],
+    },
+  },
+} as Meta<typeof AcceleratorProfiles>;
+
+const Template: StoryFn<typeof AcceleratorProfiles> = (args) => {
+  useDetectUser();
+  return <AcceleratorProfiles {...args} />;
+};
+
+export const EmptyStateNoAcceleratorProfile: StoryObj = {
+  render: Template,
+
+  parameters: {
+    msw: {
+      handlers: [
+        rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+          res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
+        ),
+        rest.get('/api/status', (req, res, ctx) => res(ctx.json(mockStatus()))),
+        rest.get(
+          '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles',
+          (req, res, ctx) => res(ctx.json(mockK8sResourceList([]))),
+        ),
```

Comment:
Why did you duplicate the mocks here? is there a reason for it

Reply:
Yes, because without duplicating, error is still there.

Reply:
I see. storybook allows you to do something like [this](https://github.com/mswjs/msw-storybook-addon#composing-request-handlers) which makes overriding better

```diff
diff --git a/frontend/src/__tests__/integration/pages/acceleratorProfiles/AcceleratorProfiles.stories.tsx b/frontend/src/__tests__/integration/pages/acceleratorProfiles/AcceleratorProfiles.stories.tsx
index 03739450..27001e57 100644
--- a/frontend/src/__tests__/integration/pages/acceleratorProfiles/AcceleratorProfiles.stories.tsx
+++ b/frontend/src/__tests__/integration/pages/acceleratorProfiles/AcceleratorProfiles.stories.tsx
@@ -13,16 +13,18 @@ export default {
   component: AcceleratorProfiles,
   parameters: {
     msw: {
-      handlers: [
-        rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
-          res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
-        ),
-        rest.get('/api/status', (req, res, ctx) => res(ctx.json(mockStatus()))),
-        rest.get(
+      handlers: {
+        status: [
+          rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+            res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
+          ),
+          rest.get('/api/status', (req, res, ctx) => res(ctx.json(mockStatus()))),
+        ],
+        accelerators: rest.get(
           '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles',
           (req, res, ctx) => res(ctx.json(mockK8sResourceList([mockAcceleratorProfile()]))),
         ),
-      ],
+      },
     },
   },
 } as Meta<typeof AcceleratorProfiles>;
@@ -37,16 +39,12 @@ export const EmptyStateNoAcceleratorProfile: StoryObj = {
 
   parameters: {
     msw: {
-      handlers: [
-        rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
-          res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
-        ),
-        rest.get('/api/status', (req, res, ctx) => res(ctx.json(mockStatus()))),
-        rest.get(
+      handlers: {
+        accelerators: rest.get(
           '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles',
           (req, res, ctx) => res(ctx.json(mockK8sResourceList([]))),
         ),
-      ],
+      },
     },
   },
 };
```